src/transformers/models/sam/processing_sam.py

```diff
@@ -51,29 +69,43 @@ class SamProcessor(ProcessorMixin):
 
     def __init__(self, image_processor):
         super().__init__(image_processor)
-        self.current_processor = self.image_processor
-        self.point_pad_value = -10
         self.target_size = self.image_processor.size["longest_edge"]
 
+    @staticmethod
+    def _add_args_for_backward_compatibility(args):
+        """
+        Remove this function once support for args is dropped in __call__
+        """
+        if len(args) > 4:
+            raise ValueError("Too many positional arguments")
+        return dict(zip(("segmentation_maps", "input_points", "input_labels", "input_boxes"), args))
+
     def __call__(
         self,
-        images=None,
-        segmentation_maps=None,
-        input_points=None,
-        input_labels=None,
-        input_boxes=None,
-        return_tensors: Optional[Union[str, TensorType]] = None,
+        images: ImageInput = None,
+        *args,  # to be deprecated
+        text: Union[TextInput, PreTokenizedInput, List[TextInput], List[PreTokenizedInput]] = None,
+        audio=None,
+        video=None,
```

Comment:
Can we also define the input time in `audio` and `video`?

Reply:
I don't understand what you mean here, could you please be more specific?

The signature of `__call__` is given, it should not be extended as far as I understand. However, it can be added as part of `VideosKwargs` and/or `AudioKwargs`. Can you point me where this attribute/variable is defined?

Reply:
*time -> *type, I think it is okay since this does not requires audio and video